### ISSUE TITLE
[modeling] Add SSD model on dev branch

### ISSUE THREAD

**Issue by hngskj** [KerasKorea/KerasObjectDetector#31]
# Add SSD model on dev branch
Github repo: [ssd_keras](https://github.com/pierluigiferrari/ssd_keras) by @pierluigiferrari

**Comment by hngskj** [comment]
# WIP : Converting a model
## Goal
`(boxes, scores, labels) = model.predict_detection('sample.jpg')`
## Working on
- [test_load_weights.py](https://github.com/hngskj/KerasObjectDetector/blob/issue_31/keras_ssd/test/test_load_weights.py)
- [keras_ssd300.py](https://github.com/hngskj/KerasObjectDetector/blob/issue_31/keras_ssd/models/keras_ssd300.py)
- [ssd_detection.py](https://github.com/hngskj/KerasObjectDetector/blob/issue_31/keras_ssd/models/ssd_detection.py)
- [keras_layer_DecodeDetections.py](https://github.com/hngskj/KerasObjectDetector/blob/issue_31/keras_ssd/keras_layers/keras_layer_DecodeDetections.py)

## Tasks
- [ ] `model = ssd_300()` is for training
- [ ] When `model.predict_detection()` is called, the `DecodeDetections` layer will be added at the end of the `model`.

**Comment by hngskj** [comment]
To run the test file, the pretrained model weights are needed. Download files from [here](https://github.com/pierluigiferrari/ssd_keras), and put them in `KerasObjectDetector/keras_ssd/pretraind/` folder.

**Comment by parkjh688** [comment]
Starting to add SSD model with @hngskj

**Comment by parkjh688** [comment]
Use the weights below.
PASCAL VOC models: 07+12: SSD300*, SSD512*
